flyctl/cmd/fly_auth_token.md

```diff
@@ -1,6 +1,6 @@
-Shows the authentication token that is currently in use.
-This can be used as an authentication token with API services,
-independent of flyctl.
+Shows the authentication token that is currently in use by flyctl.
```

Comment:
This content is extracted from the source code.

Started a new PR for this: https://github.com/superfly/flyctl/pull/3824